lib/vanagon/project.rb

```diff
@@ -476,5 +476,33 @@ def generate_package
     def generate_packaging_artifacts(workdir)
       @platform.generate_packaging_artifacts(workdir, @name, binding)
     end
+
+    # Generate a json file which lists all of the dependant components of the project
```

Comment:
This looks like the docstring for the `generate_manifest_json` method?

This method doesn't actually generate a JSON file, but a Hash, right?